ros2_utils.zsh

```diff
@@ -50,44 +50,44 @@ function rninfo {
 # Services
 
 function rslist {
-    CMD="ros2 service list"
+    CMD=(ros2 service list)
     echo $CMD
     $CMD
-    history -s rslist
-    history -s $CMD
+    print -s rslist
+    print -s $CMD
 }
```

Comment:
I guess you followed the example from rslist which was wrong from my part. Thanks for fixing!